lbrynet/db_migrator/migrate1to2.py

```diff
@@ -32,11 +36,12 @@ def migrate_blockchainname_db(db_dir):
     name_metadata = file_cursor.execute("select * from name_metadata").fetchall()
     claim_metadata = file_cursor.execute("select * from claim_ids").fetchall()
 
+    # fill n as -1, Wallet.py will be responsible for filling in correct n 
     for name, txid, sd_hash in name_metadata:
-        mem_cursor.execute("insert into name_metadata values (?, ?, ?, ?) ", (name, txid, 0, sd_hash))
+        mem_cursor.execute("insert into name_metadata values (?, ?, ?, ?) ", (name, txid, -1, sd_hash))
```

Comment:
can the `-1` be made a constant, perhaps `NOUT_UNSET`.  I dislike [magic numbers](http://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad) almost as much as grin dislikes strings.